src/components/colorPicker/index.tsx

```diff
@@ -36,13 +35,17 @@ interface Props extends ColorPickerDialogProps {
    * }
    */
   accessibilityLabels?: {
-    addButton?: string,
-    dismissButton?: string,
-    doneButton?: string,
-    input?: string
+    addButton?: string;
+    dismissButton?: string;
+    doneButton?: string;
+    input?: string;
   };
   style?: StyleProp<ViewStyle>;
   testID?: string;
+  /**
+   * Give the ColorPicker a background color
+   */
+  backgroundColor?: string;
```

Comment:
We have a 'style' prop here that we're not passing anywhere, why do we need 'backgroundColor' prop and not use the style prop for that?

Reply:
At first, I tried to use the style prop to pass the backgroundColor but I have ts error when trying to use the background prop.
```
const bg = style?.backgroundColor || Colors.$backgroundDefault;

    
Property 'backgroundColor' does not exist on type 'false | ViewStyle | RegisteredStyle<ViewStyle> | RecursiveArray<Falsy | ViewStyle | RegisteredStyle<ViewStyle>>'.
Property 'backgroundColor' does not exist on type 'false'.
```
I need to use this color for the button and pass it to the ColorPalette via style, in ColorPalette I have the same error since the containerStyle prop is the same type as the style prop in the colorPicker.

This is why I created the background prop.